contributors/devel/sig-testing/e2e-tests.md

```diff
@@ -604,6 +604,32 @@ Note that advanced testing parameters, and hierarchichally defined parameters, a
 
 In time, it is our intent to add or autogenerate a sample viper configuration that includes all e2e parameters, to ship with Kubernetes.
 
+### Pod Security Admission
+
+With introducing Pod Security admission in Kubernetes by default, it is desired to execute e2e tests within bounded pod security policy levels. The default pod security policy in e2e tests is [restricted](https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-levels). This ensures that e2e tests follow best practices for hardening pods by default.
```

Comment:
> The default pod security policy in e2e tests is restricted

Where is this set? Maybe worth mentioning in case it ever changes.

Reply:
fixed